src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/ExplicitRestoreReason.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.VisualStudio.Telemetry
+{
+    public enum ExplicitRestoreReason
+    {
+        Banner,
```

Comment:
nit: MissingPackagesBanner vs RestoreSolutionPackages? 

cc @dominoFire @chgill-MSFT for input on this.

Reply:
I like the new names, some xmldocs would explaining the origins would be good.